k8s/demo/common/sscs/sscs-tribunals-frontend.yaml

```diff
@@ -6,7 +6,7 @@ metadata:
   namespace: sscs
   annotations:
     flux.weave.works/automated: "true"
-    flux.weave.works/tag.nodejs: glob:pr-1103-*
+    flux.weave.works/tag.nodejs: glob:pr-1109-*
```

Comment:
Don't think it will work for closed PR's.
Best to set it to the same values as in the prod file

Reply:
Done, Can you please check again